Stage 3/index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz Question Form</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <h1>Quiz Question Form</h1>
+    <form id="quiz-form">
+      <label for="question">Question:</label>
+      <input type="text" id="question" maxlength="140" required />
+
+      <label for="option1">Option 1:</label>
+      <input type="text" id="option1" required />
+
+      <label for="option2">Option 2:</label>
+      <input type="text" id="option2" required />
+
+      <label for="option3">Option 3:</label>
+      <input type="text" id="option3" required />
+
+      <label for="option4">Option 4:</label>
+      <input type="text" id="option4" required />
+
+      <label for="correct-answer">Correct Answer:</label>
+      <select id="correct-answer" required>
+        <option value="1">Option 1</option>
+        <option value="2">Option 2</option>
+        <option value="3">Option 3</option>
+        <option value="4">Option 4</option>
+      </select>
+
+      <button type="button" onclick="addQuestion()">Add Question</button>
+
+      <div>
+        <h2>Quiz Questions</h2>
+        <ul id="question-list" class="question-list"></ul>
+        <button onclick="revealAnswers()">Reveal Answers</button>
+      </div>
+
+      <div>
+        <h2>Players</h2>
```

Comment:
We should be able to input the player names. Also, the players and their scores should not be visible before before we click a start game button. You can check the requirements here:

https://github.com/HackYourFuture-CPH/js-projects/blob/main/projects/quiz-app/quiz-app.md

Let me know if anything needs explaining 👍

Reply:
Hi Lucas, I understand the confusion but it was stated that I could implement one of the two or both so I chose not to take the option of user inserting their name rather than choosing the second option. But I will try to sort it out ton your liking.